frontend/src/app/_services/api/api.service.ts

```diff
@@ -519,6 +520,12 @@ export class ApiService {
       .pipe(catchError(this._handleError));
   }
 
+  /** Get All Email Log */
+  getAllEmailLog(): Observable<EmailLog[]> {
```

Comment:
This feels like it should be paginated, and potentially sorted and filtereable - which would require some updates on the frontend and backend..  Returning all the email logs every for everyone seems like a lotta-lotta stuff to return in one call and present to someone all at one.